libsolidity/interface/OptimiserSettings.h

```diff
@@ -41,10 +41,10 @@ struct OptimiserSettings
 
 			// should have good "compilability" property here.
 
-			"Tpeul"                    // Run functional expression inliner
+			"Tpeiul"                   // Run functional expression inliner
 			"xarulrul"                 // Prune a bit more in SSA
 			"xarrcL"                   // Turn into SSA again and simplify
-			"gvif"                     // Run full inliner
+			"gvPif"                    // Run full inliner
```

Comment:
The order needs to be looked at more carefully.

It would be nice if the `InlineTiny` pass can have its own name. After `p` (unusedFunctionParameterPruner), only the `InlineTiny` needs to be run.